README.md

```diff
@@ -153,7 +153,7 @@ The new version (v3) is currently under development, rapidly changing, and has n
 For the moment, this app is running in a Heroku instance with `mix phx.server`. To access the console, run:
 
 ```
-heroku run POOL_SIZE=2 iex -a <app name> -- -S mix
+heroku run FEED_STREAM_QUEUE_URL="" REDIS_URL="" POOL_SIZE=2 iex -a <app name> -- -S mix
```

Comment:
Maybe we should make this command into a mix task if it keeps getting longer? Doesn't need to be done now, just an idea